test/annealing/configuration_test.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+describe Annealing::Configuration do
```

Comment:
In order to keep it consistency with the rest of the test suite, I would like to keep the minitest sintax instead of the RSpec idiom.